src/widget/wtrackmenu.cpp

```diff
@@ -1279,6 +1283,29 @@ void WTrackMenu::slotClearBeats() {
 
 namespace {
 
+class ClearRatingTrackPointerOperation : public mixxx::TrackPointerOperation {
+  private:
+    void doApply(
+            const TrackPointer& pTrack) const override {
+        pTrack->setRating(0);
```

Comment:
Just merged (2.3/main): `TrackRecord::kNoRating`

Reply:
Looks like this is only in main. This is against 2.3 since it's very minor with no impact and adds something that should be there.

Reply:
Also in 2.3, please use

Reply:
Encapsulating this in an explicit `resetRating()` member function in `Track` would be even better.